test/cctest/test_url.cc

```diff
@@ -136,15 +136,15 @@ TEST_F(URLTest, FromFilePath) {
 #ifdef _WIN32
   file_url = URL::FromFilePath("C:\\Program Files\\");
   EXPECT_EQ("file:", file_url.protocol());
-  EXPECT_EQ("/C:/Program%20Files/", file_url.path());
+  EXPECT_EQ("//C:/Program%20Files/", file_url.path());
 
   file_url = URL::FromFilePath("C:\\a\\b\\c");
   EXPECT_EQ("file:", file_url.protocol());
-  EXPECT_EQ("/C:/a/b/c", file_url.path());
+  EXPECT_EQ("//C:/a/b/c", file_url.path());
```

Comment:
I'm a little confused by the additional `/` returned by `path()`, as it doesn't seem to match the fixtures for web platform tests:

https://github.com/web-platform-tests/wpt/blob/master/url/resources/urltestdata.json

I'm not an expert in this area, so let me know if I'm just misunderstanding the API.

---

Something that also jumps out at me, that I find a bit confusing is that tests for URLs that include a drive letter seem to have been [dropped from the suite](https://github.com/web-platform-tests/wpt/commit/050308a616a8388f1ad5d6e87eac0270fd35023f#diff-760122a1234231092e5a1a2ca8655bffdda80ff57f58327f69823cf31423c39aL5632), and they've been swapped out for tests fixtures of the form:

```
file://host/C:
```

Does this mean that some of the file patterns we've written tests for are technically no longer supported?

Reply:
For path changed, the spec is updated to add a new normalization and some tests matched with the update use `//`. Refs: https://github.com/web-platform-tests/wpt/commit/050308a616a8388f1ad5d6e87eac0270fd35023f#diff-760122a1234231092e5a1a2ca8655bffdda80ff57f58327f69823cf31423c39a.

Reply:
The test suite is about `whatwg-url` module and we pull its tests from WPT so that I think the file patterns we've written is not affected except `cctest/node_url.cc` that we updated to follow WPT.
> Does this mean that some of the file patterns we've written tests for in Node.js, are technically no longer supported (and have potentially undefined behavior?).